jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -87,31 +87,24 @@ function configure_router_location {
     local routerUrlExternal="${KIE_SERVER_ROUTER_URL_EXTERNAL}"
     local defaultInsecureHost="${HOSTNAME_HTTP:-${HOSTNAME:-localhost}}"
     local defaultSecureHost="${HOSTNAME_HTTPS:-${defaultInsecureHost}}"
-
     if [ "${host}" = "" ]; then
         host=$(find_env "${routeService}_SERVICE_HOST")
     fi
     JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.router.host=${host}"
     if [ "${port}" = "" ]; then
         port=$(find_env "${routeService}_SERVICE_PORT" "9000")
     fi
-
     JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.router.port=${port}"
     if [ -z "${routerUrlExternal}" ]; then
        if [ -n "${routeName}" ]; then
             if [ "${protocol}" = "https" ]; then
-                routeName="${routeName}"
-                host="${host:-${defaultSecureHost}}"
+                routeName="secure-${routeName}"
                 port="${port:-443}"
             else
                 protocol="${protocol:-http}"
-                host="${host:-${defaultInsecureHost}}"
                 port="${port:-80}"
             fi
-
-	        local routeHost=$(query_route_host "${routeName}" "${host}:${port}")
-	        routerUrlExternal="${protocol}://${routeHost}"
-
+            routerUrlExternal=$(build_route_url "${routeName}" "${protocol}" "${host}" "${port}")
```

Comment:
be careful to do not re-introduce the issue fixed by: https://github.com/jboss-container-images/jboss-kie-modules/commit/6b90b305106f10baea729c4e67687bfa3f694c92